src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -123,6 +130,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('acr build') as c:
         c.argument('registry_name', options_list=['--registry', '-r'])
         c.positional('source_location', help="The local source code directory path (e.g., './src') or the URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') or a remote tarball (e.g., 'http://server/context.tar.gz').", completer=FilesCompleter())
+        c.argument('no_push', help="Indicates whether the image built should be pushed to the registry.", action='store_true')
```

Comment:
This is essentially a breaking change since the `--no-push true` will cease to be functional. If so, please add an entry in the `HISTORY.rst` to call out the breaking change.

Reply:
Yes, updated history, right now `--no-push .` is not working.